integration-tests/src/lib.rs

```diff
@@ -217,10 +214,7 @@ pub async fn test_two_apps<A1, A2, F>(
     )
     .await
     .unwrap();
-    let (res1, res2) = tokio::join!(
-        first_handle.stop_and_wait(APP_STOP_TIMEOUT),
-        second_handle.stop_and_wait(APP_STOP_TIMEOUT),
-    );
+    let (res1, res2) = tokio::join!(first_handle.stop_and_wait(), second_handle.stop_and_wait(),);
```

Comment:
:art: trailing comma not needed anymore
```suggestion
    let (res1, res2) = tokio::join!(first_handle.stop_and_wait(), second_handle.stop_and_wait());
```